conbench/tests/api/test_runs.py

```diff
@@ -231,6 +231,27 @@ def test_closest_commit_different_machines_should_not_prefer_test_runs_as_baseli
         response = client.get(f"/api/runs/{contender_run.id}/")
         self.assert_200_ok(response, _expected_entity(contender_run, baseline_run.id))
 
+    def test_baseline_run_always_on_default_branch(self, client):
+        self.authenticate(client)
+        _, benchmark_results = _fixtures.gen_fake_data()
+
+        successes = 0
+        for benchmark_result in benchmark_results:
+            response = client.get(f"/api/runs/{benchmark_result.run_id}/")
+            assert response.status_code == 200, response.status_code
```

Comment:
unrelated, but I want to say this here because I keep "fixing" this in our codebase. Let's write these as

```
assert response.status_code == 200, response.text
```
Because when we get an unexpected response what we usually want to see in the output (if we have to pick _one_ thing) is the response body. We already know that the status code isn't the one we expected, and so to get to see some error detail we usually want to peek into the body.

Reply:
Changed to this 👀 
```py
assert response.status_code == 200, f"bad response: {response.__dict__}"
```

Reply:
:+1: